go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/emirpasic/gods
+
+go 1.13
```

Comment:
It's a better idea to use the latest version (`v1.14`)

Reply:
the go version is the minimal go version GoDS can run on. I am sure GoDS runs even on the first Go version, but I have stated 1.2 in the go.mod